code/modules/clothing/neck/_neck.dm

```diff
@@ -180,13 +180,19 @@
 	icon_state = "stripedbluescarf"
 	item_color = "stripedbluescarf"
 
-/obj/item/clothing/neck/petcollar //don't really wear this though please c'mon seriously guys
+/obj/item/clothing/neck/petcollar
 	name = "pet collar"
-	desc = "It's for pets. Though you probably could wear it yourself, you'd doubtless be the subject of ridicule."
+	desc = "It's for pets."
 	icon_state = "petcollar"
 	item_color = "petcollar"
 	var/tagname = null
 
+/obj/item/clothing/neck/petcollar/mob_can_equip(mob/M, mob/equipper, slot, disable_warning = 0)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.dna?.species?.id == "human")
```

Comment:
remove 92&3

Reply:
Am I misinterpreting ShizCalev's review or is it merely incorrect?

Reply:
He probably forgot we already had species helpers.

Reply:
species helpies